legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/DataSpaceAnalyticsArtifactGenerationExtension.java

```diff
@@ -27,33 +31,28 @@
 import org.finos.legend.pure.generated.Root_meta_pure_metamodel_dataSpace_DataSpace;
 import org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.PackageableElement;
 
-import java.util.Collections;
-import java.util.List;
-
-import static org.finos.legend.engine.language.pure.compiler.toPureGraph.HelperModelBuilder.getElementFullPath;
-import static org.finos.legend.pure.generated.platform_pure_corefunctions_meta.Root_meta_pure_functions_meta_elementToPath_PackageableElement_1__String_1__String_1_;
-
 public class DataSpaceAnalyticsArtifactGenerationExtension implements ArtifactGenerationExtension
 {
+    public final String ROOT_PATH = "analytics";
 
-    public final String ROOT_PATH = "data-space-analytics";
     public static ObjectMapper objectMapper = ObjectMapperFactory.getNewStandardObjectMapperWithPureProtocolExtensionSupports();
 
     @Override
-    public boolean canGenerate(PackageableElement element)
+    public String getName()
     {
-        return element instanceof Root_meta_pure_metamodel_dataSpace_DataSpace;
+        return "DataSpace Analytics";
     }
 
     @Override
-    public String getArtifactsRootPath()
+    public boolean canGenerate(PackageableElement element)
     {
-        return ROOT_PATH;
```

Comment:
Why don't we enforce a non-empty root path? This could make things a little more organized